src/Stratis.Bitcoin.IntegrationTests/APITests.cs

```diff
@@ -89,40 +66,35 @@ public void CanGetGeneralInfoViaAPI()
         [Fact]
         public void CanStartStakingViaAPI()
         {
-            try
+            using (NodeBuilder builder = NodeBuilder.Create())
             {
-                var fullNode = this.apiTestsFixture.stratisStakeNode.FullNode;
-                var apiURI = fullNode.NodeService<ApiSettings>().ApiUri;
+                var posNode = builder.CreateStratisPosApiNode();
+                posNode.Start();
 
-                Assert.NotNull(fullNode.NodeService<IPosMinting>(true));
+                var apiURI = posNode.FullNode.NodeService<ApiSettings>().ApiUri;
```

Comment:
Can move this line close to where apiURI is being used.  Something like 

![image](https://user-images.githubusercontent.com/16000683/38771965-6973ee9a-4024-11e8-875d-b450249a8a93.png)

Reply:
Fixed